src/components/Coaching/LoadCoachingList.graphql

```diff
@@ -0,0 +1,35 @@
+query LoadCoachingList {
+  coachingAccountLists {
+    nodes {
+      ...Coach
+    }
+    totalCount
+    totalPageCount
+    pageInfo {
+      startCursor
+      endCursor
+      hasNextPage
+      hasPreviousPage
+    }
+  }
+}
+
+fragment Coach on CoachingAccountList {
```

Comment:
Should this be named Coachee or CoachedPerson or CoachedAccountList or something? Idk what the correct word is. But it's a list of people being coached, not coaches right?